### ISSUE TITLE
Corruption with column selection + moving lines

### ISSUE THREAD

**Issue by Watilin** [notepad-plus-plus/notepad-plus-plus#4093]
### Description of the Issue

While having a column selection active, when moving lines with `Ctrl+Shift+Up` or `Ctrl+Shift+Down`, scrambled, seemingly random characters appear.

### Steps to Reproduce the Issue

1. Open a new buffer
2. Type in the following:
> The quick brown fox
> jumps over
> the lazy dog.
3. Now the caret should be at the end of the last line. Hit `Shift+Alt+Up` to make a column selection to the previous line. A rectangular selection should appear.
4. Hit `Ctrl+Shift+Up` to moves lines towards the top.

### Expected Behavior

In my opinion, the selected lines should have moved up just like lines selected the “regular” (not column) fashion. I think the result should be:
> jumps over
> the lazy dog.
> The quick brown fox

### Actual Behavior
Now the text looks similar to this:
>
> og.
> `NUL` `NUL` `NUL` `xDE` `NUL` `NUL`The quick brown fox
> jumps over
> the lazy d

### Debug Information

```
Notepad++ v7.5.3   (32-bit)
Build time : Dec  5 2017 - 10:45:46
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 7 (32-bit)
Plugins : CodeAlignmentNpp.dll ComparePlugin.dll HTMLTag_unicode.dll JSLintNpp.dll mimeTools.dll NppConverter.dll NppEditorConfig.dll NppExport.dll NppFTP.dll NppSaveAsAdmin.dll PluginManager.dll 
```

**Comment by sasumner** [comment]
Wow.  Good find!  To be clear, why don't you add in your Expected Behavior section what the final text SHOULD look like, in your opinion.

**Comment by dail8859** [comment]
Duplicate of #2007, #2928

**Comment by Watilin** [comment]
> To be clear, why don't you add in your Expected Behavior section what the final text SHOULD look like, in your opinion
I edited my first message ;)

**Comment by dail8859** [comment]
The text corruption is fixed in Notepad v7.7 since it upgraded to a new version of Scintilla.

*Please note*: The fix does not give you the expected results of moving the lines (it simply just doesn't work with a rectangular selection)...this is something that would need addressed by the Scintilla project.
